modules/ROOT/pages/upgrade-service-mesh.adoc

```diff
@@ -0,0 +1,60 @@
+= Upgrade Service Mesh
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+:keywords: service mesh, microservices, upgrading
+
+
+Anypoint Service Mesh v1.1 includes new features and architecture enhancements, such as bypassing the Istio Mixer when sending requests from clients to Anypoint Platform. This change caused an Anypoint Service Mesh architecture updates, and therefore, introduces a downtime for the Anypoint Service Mesh upgrade to v1.1.
+
+Anypoint Service Mesh v1.1 includes new features and architecture enhancements, such as stopping using Istio Mixer. Because of this change, upgrading from Anypoint Service Mesh v1.0 to v1.1 introduces downtime during the c
+
+== Prerequisites
+
+Before you upgrade, ensure that you meet the following requirements:
+
```

Comment:
I expect a list of prerequisites to be just that: a list of conditions that exist, but not **how** to achieve them; those are separate tasks covered in other topics, presumably.
You probably do not have time to do this now, but I suggest altering the list as follows:
```suggestion
Before you upgrade, ensure that your environment uses:
* Anypoint Service Mesh v1.0.x.
* The most recent version of the Anypoint Service Mesh command line
* Istio 1.6.0 or later for both control plane and data plane
```
RE the second bullet, note that your later text specifies v1.1.x rather than "latest": clarify.

Reply:
You are right! Thanks! Done!